Add simple RSA encode/decode example in Cryptography

BlueCocoa opened the pull request:
A simple RSA example that uses public key to encode message and private key to decode message.

Signed-off-by: BlueCocoa 0xbbc@0xbbc.com

BlueCocoa committed:
Add simple RSA encode/decode example in Cryptography

A simple RSA example that uses public key to encode message and private key to decode message.

Signed-off-by: BlueCocoa <0xbbc@0xbbc.com>

review-ninja commented:
<a href="https://app.review.ninja/Llama-Group/Project-Llama/pull/22" target="_blank"><img src="https://app.review.ninja/63898156/pull/22/badge" alt="ReviewNinja"/>

BlueCocoa committed:
Add rsa test with travis

Add rsa test in .travis.yml

Signed-off-by: BlueCocoa <0xbbc@0xbbc.com>

BlueCocoa committed:
Modified namespace for Cryptography.h

Seperated namespace instead of using c++1z extension 'namespace alpaca::cryptography'.

Signed-off-by: BlueCocoa <0xbbc@0xbbc.com>

BlueCocoa committed:
Added libcrypto to all examples

Added libcrypto to all examples.

Signed-off-by: BlueCocoa <0xbbc@0xbbc.com>
